uk.co.compucorp.civicrm.hrcontactactionsmenu/templates/CRM/ContactActionsMenu/Page/Inline/ActionsPart1.tpl

```diff
@@ -0,0 +1,90 @@
+<div class="ac_results" id="crm-contact-actions-list">
+  <div id="bootstrap-theme">
+    <div class="row show-grid">
+      <div class="col-md-4 first-section">
+        <h3>User Informations:</h3>
+        <p>There is no user for this staff member</p>
+        <button type="button" class="btn btn-primary">
+          <span class="btn-icon">
+            <i class="fa fa-plus"></i>
+          </span> 
+          Create a user for this staff member
+        </button>
+      </div>
+      <div class="col-md-8 second-section">
+        <div class="row show-grid">
+          <div class="col-md-4">
+            <h3>Leave:</h3>
+            <p>
+              <button type="button" class="btn btn-primary-outline">
+                <span class="btn-icon"><i class="fa fa-plus"></i></span> Record New Leave
+              </button>
+            </p>
+            <p>
+              <button type="button" class="btn btn-primary-outline">
+                <span class="btn-icon"><i class="fa fa-search"></i></span> View Entitlements
+              </button>
+            </p>
+            <hr class="divider">
+            <p>You have not selected a leave approver</p>
```

Comment:
I can see you used `p` tag in many places, why exactly do we need them, why cant we use `div` or `span`.

Reply:
I have used `p` instead of `div/span` because `p` already hide bottom margin of `10px` which is suited for design.  If we use div an additional class with bottom margin of `10px` is required. What do you suggest?

Reply:
In that its better to use `div`, because `p` is not meant to be used like this, as it is only for paragraphs.